packages/shared/src/convert-to-className.js

```diff
@@ -58,7 +58,9 @@ export function convertStyleToClassName(
 
   // NOTE: '<>' is used to keep existing hashes stable.
   // This should be removed in a future version.
-  const className = classNamePrefix + createHash('<>' + stringToHash);
+  const className = debug
+    ? `${dashedKey}-${classNamePrefix}${createHash('<>' + stringToHash)}`
```

Comment:
I think it would be slightly clearer if we don't use the dashedKey here, as it will match the property name in the source code. We also wouldn't typically introduce dashes into the var names, which we'll want give the same treatment when debug is true.

Reply:
```suggestion
    ? `${key}-${classNamePrefix}${createHash('<>' + stringToHash)}`
```